openff/toolkit/tests/test_forcefield.py

```diff
@@ -931,6 +931,25 @@ def test_force_field_case(self):
 
         assert hash(default_case) == hash(lower_case)
 
+    def test_do_not_load_in_child_dir(self):
```

Comment:
Could we do this test in a temp dir? Right now it leaves the directories around.
```suggestion
    def test_do_not_load_in_child_dir(self, tmp_path):
```

Reply:
Good catch - I wonder how many other times our tests leave temporary directories/files/souvenirs behind?

Reply:
Hmm, seems like that didn't quite do it:

```
$ # git clone https://github.com/openforcefield/openff-toolkit
$ # cd openff-toolkit
$ # conda activate openff-dev
$ # python -m pytest -v -r fE --tb=short --ignore=openff/toolkit/tests/test_examples.py  --ignore=openff/toolkit/tests/test_links.py --runslow openff/toolkit/tests
$ git status
git status                                                                                                                                         master  ✱
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	a/
	openff/toolkit/data/molecules/AlkEthOH_tripos/
	openff/toolkit/data/molecules/FreeSolv/
```

Reply:
Hm, I don't remember the exact convention with this, but we may need to have a context manager `temporary_cd` into that directory as well.

Reply:
Yeah, just creating the tmp_path doesn't go to it. I use this construction a lot:

```python
def testy_test(tmpdir):
	with tmpdir.as_cwd():
		print(os.cwd())
```